tests/specs/install/future_install_global/__test__.jsonc

```diff
@@ -0,0 +1,16 @@
+{
+  "tempDir": true,
+  "envs": {
+    "DENO_FUTURE": "1"
+  },
+  "steps": [
+    {
+      "args": "install --global --root ./bins --name deno-test-bin ./main.js",
+      "output": "install.out"
+    },
+    {
+      "args": "run -A assert.js",
+      "output": "assert.out"
```

Comment:
Instead of this, change to:

```json
  "output": ""
```

...then delete `assert.out`.

The "output" property will use the text in the string when it doesn't end in `.out`